ml-algorithms/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     implementation group: 'io.protostuff', name: 'protostuff-collectionschema', version: '1.8.0'
     testImplementation group: 'junit', name: 'junit', version: '4.13.2'
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '5.3.1'
+    testImplementation group: 'org.mockito', name: 'mockito-inline', version: '4.4.0'
```

Comment:
Not against using this version, but is there a reason for using '4.4.0' rather than the latest '5.x.x' version?

Reply:
I believe `mockito-core` should already have this `mockito-inline` ability in `5.3.1` version. Could you please check?

Reply:
This was present in `agent_framework_dev` branch, will check if this dependency needs to be explicitly added.